src/hooks/usePolicy.ts

```diff
@@ -0,0 +1,17 @@
+import {useOnyx} from 'react-native-onyx';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+function usePolicy(policyID?: string) {
+    const getPolicyID = () => {
```

Comment:
This can be a pure function defined outside of the hook, so let's move it there so it isn't re-defined every time this hook is used:

```ts
function getPolicyIDOrDefault(policyID?: string) {
    if (!policyID || policyID === CONST.POLICY.OWNER_EMAIL_FAKE) {
        return '-1';
    }
    return policyID;
}

function usePolicy(policyID?: string) {
    const [policy] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY}${getPolicyIDOrDefault(policyID)}`);
    return policy;
}
```